src/Package/Repositories/Statuses.php

```diff
@@ -95,4 +95,24 @@ public function getList($user, $repo, $sha)
 		// Send the request.
 		return $this->processResponse($this->client->get($this->fetchUrl($path)));
 	}
+
+	/**
+	 * Method to get the combined Status for a specific Ref.
+	 *
+	 * @param   string  $user  The name of the owner of the GitHub repository.
+	 * @param   string  $repo  The name of the GitHub repository.
+	 * @param   string  $sha   SHA1 for which to get the combined status.
+	 *
+	 * @return  array
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getCombinedStatus($user, $repo, $sha)
+	{
+		// Build the request path.
+		$path = '/repos/' . $user . '/' . $repo . '/commits/' . $sha . '/status';
```

Comment:
If there's something in the coding standard about it, I'd probably rewrite or remove that section in full.  IMO there's no need to do excessive string concatenation over simple string variables like this.  All that to say we agree 😆